tests/fast/Watches.toml

```diff
@@ -3,3 +3,6 @@ testTitle = 'WatchesTest'
 
     [[test.workload]]
     testName = 'Watches'
+
+    [[test.workload]]
+    testName = 'WatchesSameKeyCorrectness'
```

Comment:
Set `numWatches` here, even if just to the default value, to make it clear it can be changed.